pulser/register.py

```diff
@@ -96,6 +99,9 @@ def rectangle(cls, rows: int, columns: int, spacing: float = 4.0,
             prefix (str): The prefix for the qubit ids. If defined, each qubit
                 id starts with the prefix, followed by an int from 0 to N-1
                 (e.g. prefix='q' -> IDs: 'q0', 'q1', 'q2', ...)
+
+        Returns:
+            Register: A register with qubits placed in a rectangular way.
```

Comment:
nit: I "rectangular array" might be more appropriate. The same goes for the other docstrings.